### DISCUSSION TITLE
[Help Wanted] Can’t access my Protected Notes, any way to reset my password?

### DISCUSSION THREAD

**Discussion by Viloz32** [standardnotes/forum#2748] [General] [answered]
**Describe the issue**
A clear and concise description of what the issue is.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by effieeee** [comment, marked as answer]
Hi there, if you're still signed into the app on any device, you can set a **passcode lock** there, and the protected note will require the passcode lock instead. On the desktop or web app, you can find this option in the **Account** menu on the lower left, while on the mobile app, it's located in the **Settings** menu.

I'm afraid we're really unable to reset passwords though, so if you're really unable to recall your account password, please refer to this help post for instructions on how to move to a new account, since you're still signed into your current one:
https://standardnotes.com/help/6/i-ve-forgotten-my-password-what-should-i-do

**Comment by Viloz32** [comment]
> Hi there, if you're still signed into the app on any device, you can set a **passcode lock** there, and the protected note will require the passcode lock instead. On the desktop or web app, you can find this option in the **Account** menu on the lower left, while on the mobile app, it's located in the **Settings** menu.
> 
> 
> 
> I'm afraid we're really unable to reset passwords though, so if you're really unable to recall your account password, please refer to this help post for instructions on how to move to a new account, since you're still signed into your current one:
> 
> https://standardnotes.com/help/6/i-ve-forgotten-my-password-what-should-i-do

Honestly, am short of words. May knowledge and understanding never depart from you, that's my little prayer for you dear. Thanks soo much
